README.md

```diff
@@ -85,22 +145,61 @@ this does not mean that datasets must be curated with both "forward" (`u` to
 `v`) Edges and "reverse" (`v` to `u`) Edges: any "reverse" edge can be
 inferred during graph creation.
 
-#### Points
+#### Zones
+[Zones](#zones) are polygon features that also contain metadata to identify
+them as network (graph) edges. Currently, this means that they must have a list
+of node references: `_w_id`, which mean "this 2-dimensional polygon feature consists
+of a complete graph with every pair of distinct nodes in `_w_id` connected by a unique edge.
+Note that this would yield k(k-1)/2 edges for a zone comprised of k nodes.
```

Comment:
```suggestion
Note that this would yield $k(k-1)/2$ edges for a zone comprised of $k$ nodes.
```

Purely cosmetic... some fancy math formatting since you made the effort to express it as such 😄

Rendered:

> Note that this would yield $k(k-1)/2$ edges for a zone comprised of $k$ nodes.